src/rpc/governance.cpp

```diff
@@ -0,0 +1,1021 @@
+
+    // Get first superblock
+    int nFirstSuperblockOffset = (nSuperblockCycle - nSuperblockStartBlock % nSuperblockCycle) % nSuperblockCycle;
+    int nFirstSuperblock = nSuperblockStartBlock + nFirstSuperblockOffset;
+
+    if(nBlockHeight < nFirstSuperblock){
+        nLastSuperblock = 0;
+        nNextSuperblock = nFirstSuperblock;
+    } else {
+        nLastSuperblock = nBlockHeight - nBlockHeight % nSuperblockCycle;
+        nNextSuperblock = nLastSuperblock + nSuperblockCycle;
+    }
+
+    UniValue obj(UniValue::VOBJ);
+    obj.push_back(Pair("governanceminquorum", Params().GetConsensus().nGovernanceMinQuorum));
+    obj.push_back(Pair("masternodewatchdogmaxseconds", MASTERNODE_WATCHDOG_MAX_SECONDS));
+    obj.push_back(Pair("proposalfee", ValueFromAmount(GOVERNANCE_PROPOSAL_FEE_TX)));
+    obj.push_back(Pair("superblockcycle", Params().GetConsensus().nSuperblockCycle));
+    obj.push_back(Pair("lastsuperblock", nLastSuperblock));
+    obj.push_back(Pair("nextsuperblock", nNextSuperblock));
+    obj.push_back(Pair("maxgovobjdatasize", MAX_GOVERNANCE_OBJECT_DATA_SIZE));
+
+    return obj;
+}
+
+UniValue getsuperblockbudget(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 1) {
+        throw std::runtime_error(
+                    "getsuperblockbudget index\n"
+                    "\nReturns the absolute maximum sum of superblock payments allowed.\n"
+                    "\nArguments:\n"
+                    "1. index         (numeric, required) The block index\n"
+                    "\nResult:\n"
+                    "n                (numeric) The absolute maximum sum of superblock payments allowed, in " + CURRENCY_UNIT + "\n"
+                                                                                                                                "\nExamples:\n"
+                    + HelpExampleCli("getsuperblockbudget", "1000")
+                    + HelpExampleRpc("getsuperblockbudget", "1000")
+                    );
+    }
+
+    int nBlockHeight = request.params[0].get_int();
+    if (nBlockHeight < 0) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Block height out of range");
+    }
+
+    CAmount nBudget = CSuperblock::GetPaymentsLimit(nBlockHeight);
+    std::string strBudget = FormatMoney(nBudget);
+
+    return strBudget;
+}
+
+static const CRPCCommand commands[] =
+{ //  category              name                      actor (function)         argNames
+  //  --------------------- ------------------------  -----------------------  ----------
+  { "governance",             "gobject",       &gobject,       {"",""} },
+  { "governance",             "voteraw",          &voteraw,          {} },
+  { "governance",             "getgovernanceinfo",  &getgovernanceinfo,  {} },
+  { "governance",             "getsuperblockbudget",       &getsuperblockbudget,       {"index"} },
+};
+
+void RegisterMiningRPCCommands(CRPCTable &t)
```

Comment:
wrong name, apparently this function is not declared as extern and it's not used.